keyboards/melgeek/mj61/mj61.c

```diff
@@ -0,0 +1,113 @@
+};
+
+led_config_t g_led_config = {
+	{
+		/* C0     C1         C2        C3        C4        C5        C6        C7        C8          C9        C10         C11        C12        C13  */
+		{  0,      1,         2,        3,        4,        5,        6,        7,        8,          9,        10,         11,       12,        13     }, /* R0 */
+		{ 27,     26,        25,       24,       23,       22,       21,       20,       19,         18,        17,         16,       15,        14     }, /* R1 */
+		{ 28,     29,        30,       31,       32,       33,       34,       35,       36,         37,        38,         39,       40,        NO_LED,}, /* R2 */
+		{ 52,     NO_LED,    51,       50,       49,       48,       47,       46,       45,         44,        43,         42,       41,        NO_LED }, /* R3 */
+		{ 53,     54,        NO_LED,   55,       61,       56,       62,   NO_LED,   NO_LED,         57,        58,         59,       60,        NO_LED }, /* R4 */
+    }, {
+        {0, 0},   {16, 0},   {32, 0},  {48, 0},  {64, 0},  {80, 0},  {96, 0},  {112, 0},  {128, 0},  {144, 0},  {160, 0},  {176, 0},  {192, 0},  {208, 0},
+		{0, 16},  {16, 16},  {32, 16}, {48, 16}, {64, 16}, {80, 16}, {96, 16}, {112, 16}, {128, 16}, {144, 16}, {160, 16}, {176, 16}, {192, 16}, {208, 16},
+		{0, 32},  {16, 32},  {32, 32}, {48, 32}, {64, 32}, {80, 32}, {96, 32}, {112, 32}, {128, 32}, {144, 32}, {160, 32}, {176, 32}, {192, 32},
+		{0, 48},             {16, 48}, {32, 48}, {48, 48}, {64, 48}, {80, 48}, {96, 48},  {112, 48}, {128, 48}, {144, 48}, {160, 48}, {176, 48},
+		{0, 64},  {16, 64},            {32, 64}, {48, 64}, {64, 64}, {80, 64},                       {96, 64},  {112, 64}, {128, 64}, {144, 64},
+	}, {
+	       4,      4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4, 
+		   4,      4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4,
+		   4,      4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,
+		   4,                 4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,
+		   4,      4,                   4,        4,        4,        4,                              4,         4,          4,        4,
+	}
+};
+
+void matrix_init_kb(void) {
+    matrix_init_user();
+}
+
+void matrix_scan_kb(void) { matrix_scan_user(); }
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) { return process_record_user(keycode, record); }
+
+void suspend_power_down_kb(void) {
+    rgb_matrix_set_suspend_state(true);
+    suspend_power_down_user();
+}
+
+void suspend_wakeup_init_kb(void) {
+    rgb_matrix_set_suspend_state(false);
+    suspend_wakeup_init_user();
+}
+
+void matrix_setup(void) {
+	palSetPadMode(GPIOA, 2, PAL_MODE_OUTPUT_PUSHPULL);
+	palSetPad(GPIOA, 2);
+}
```

Comment:
Is there a specific need for this? 

Also:
```suggestion
void keyboard_pre_init_kb(void) {
    setPinOutput(A2);
    writePinHigh(A2);
    keyboard_pre_init_user();
}

Reply:
> Is there a specific need for this?
> 
> Also:

Yes. GPIOA 2 is the power control of ISSI3741.